spec/tasks/reactions_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'highline/import'
+require 'highline/simulate'
+
+RSpec.describe "reactions tasks" do
+  fab!(:user_1) { Fabricate(:user) }
+  fab!(:user_2) { Fabricate(:user) }
+  fab!(:user_3) { Fabricate(:user) }
+  fab!(:user_4) { Fabricate(:user) }
+  fab!(:post_2) { Fabricate(:post, user: user_1) }
+  fab!(:reaction_1) { Fabricate(:reaction, post: post_2, reaction_value: "laughing") }
+  fab!(:reaction_2) { Fabricate(:reaction, post: post_2, reaction_value: "open_mouth") }
+  fab!(:reaction_3) { Fabricate(:reaction, post: post_2, reaction_value: "hugs") }
+  fab!(:reaction_user_1) { Fabricate(:reaction_user, reaction: reaction_1, user: user_2, post: post_2) }
+  fab!(:reaction_user_2) { Fabricate(:reaction_user, reaction: reaction_1, user: user_1, post: post_2) }
+  fab!(:reaction_user_3) { Fabricate(:reaction_user, reaction: reaction_3, user: user_4, post: post_2) }
+  fab!(:reaction_user_4) { Fabricate(:reaction_user, reaction: reaction_2, user: user_3, post: post_2) }
+
+  before do
+    Rake::Task.clear
+    Discourse::Application.load_tasks
```

Comment:
@jjaffeux I am stuck here, `load_tasks` doesn't load the plugin's rake tasks. I am not getting a way to load it.

It gives an error
```
RuntimeError:
       Don't know how to build task 'reactions:nuke'
```

Reply:
try to manually `load` the task file at the top of your file

Reply:
I loaded the rake file in the plugin.rb & it worked 😅